src/main/java/edu/ucla/library/prl/harvester/services/HarvestJobSchedulerServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+
+package edu.ucla.library.prl.harvester.services;
+
+import edu.ucla.library.prl.harvester.Job;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * The interface of the event bus service that schedules harvest jobs.
+ */
+@SuppressWarnings("PMD.UnusedFormalParameter") //temp until constructor defined
```

Comment:
How obvious it is that this should be removed at some point is debatable, but I feel like I'd be more likely to catch it if it's marked like:

```suggestion
@SuppressWarnings("PMD.UnusedFormalParameter") // FIXME: temp until constructor defined
```

Reply:
I tend to look at all @SuppressWarnings as technical debt, but fine with FIXMEs that suggest we are planning on doing this sooner rather than later. It adds a sense of urgency to it.